scripts/analysis.py

```diff
@@ -720,9 +825,11 @@ def stacked_bar_chart(dictionary, timestep,
         if isinstance(key, str) is True:
             label = key.replace('_government', '')
         else:
-            label = str(nucname.name(key))
+            label = str(key)
         # very first country does not have a 'bottom' argument
-        if top_index is True:
+        if "Sink" in label:
+            print("Ignore Sink Institution")
```

Comment:
this is addressed in [here](https://github.com/arfc/transition-scenarios/pull/42/commits/10bf9f1a381b3ed0b93fad31027c390952989829)